kubeflow/notebooks/notebook.example.yaml

```diff
@@ -0,0 +1,86 @@
+{
+  "apiVersion": "kubeflow.org/v1alpha1",
+  "kind": "Notebook",
+  "metadata": {
+    "annotations": {
+      "accessLocalFs": "false",
+      "disks": "null",
+      "image": "tensorflow-1.10.1-notebook-cpu:v0.3.0",
```

Comment:
why are all these necessary?

Reply:
please ignore. They were hold-overs when I was fairly early in to implementation.